logic/src/commonMain/kotlin/com/wire/kalium/logic/data/event/Event.kt

```diff
@@ -302,6 +302,23 @@ sealed class Event(open val id: String, open val transient: Boolean) {
                 senderUserIdKey to senderUserId.toLogString(),
             )
         }
+
+        data class ConversationMessageTimer(
+            override val id: String,
+            override val conversationId: ConversationId,
+            override val transient: Boolean,
+            val messageTimer: Long?,
```

Comment:
Why don't we declare value as `Duration` type for coherence with the other duration/timeouts values?
```suggestion
            val messageTimerDuration: Duration,
```